### DISCUSSION TITLE
Suggestion: location of relationship verbalizaton

### DISCUSSION THREAD

**Discussion by robbebob77** [Hoogkamer/Termboard#21] [Q&A] [answered]
I really like your Termboard! A suggestion: would it be possible to position the text of relationships above or below the relationship line? Now it is on top of that line, making the verbalizations on those relationship arrows sometimes hard to read.

Okay, two more suggestions in the same area:
* Would be nice if the text size of relationships is adjustable, just like the size of terms
* Sometimes there is no direction in a relationship; or it is bi-directional. Would be nice if there was a possibility to have no arrow-tip at all respectively arrow tips on both sides of the relationship

**Comment by Hoogkamer** [comment]
Hi thank you! And also for your feedback and your example graph. 
I see the issue with the relation labels being hard to read in your graph. I solved that by giving the label a white background
And your other 2 points are added now, so you can try them out.

Regards

**Reply by Hoogkamer** [reply, marked as answer]
In the top toolbar you can set the size (and fontsize) of relations. Also you can select if you want to have 1, 2 or zero arrows
